artifactory_cleanup/cli.py

```diff
@@ -184,9 +195,9 @@ def main(self):
                     "name": summary.policy_name,
                     "file_count": summary.artifacts_removed,
                     "size": summary.artifacts_size,
+                    **({"removed_artifacts_list": summary.removed_artifacts_list} if summary.removed_artifacts_list is not None else {})
```

Comment:
> **({"removed_artifacts_list": summary.removed_artifacts_list} if summary.removed_artifacts_list is not None else {})

I don't like that constrcution - too much actions in a single line

Please make it simplier by using additional variables. 
We can always add `artifacts` key, but just set it to `null` if saving not required

Reply:
at the end it should be like 
```suggestion
                    "artifacts": summary.removed_artifacts
```